pages/common/topydo.md

```diff
@@ -0,0 +1,28 @@
+# topydo
+
+> A powerful todo list application using the todo.txt format.
+> More information: <https://github.com/topydo/topydo>.
+
+- Add todo to the project "tldr" and the context "github":
+
+`topydo add "merge pull request +tldr @github"`
+
+- Add todo which should be done till tomorow with a "A" priority:
+
+`topydo add "(A) call mom due:1d"`
```

Comment:
Perhaps this will read better?
```suggestion
- Add a todo with a due date of tomorrow with a priority of `A`:

`topydo add "(A) {{todo _message}} due:{{1d}}"`
```

Are there any other priorities here we could add? We could use the syntax `{{foo|bar|baz}}` for example.